src/select/Select.ts

```diff
@@ -18,382 +17,332 @@ import * as iconCss from '../common/styles/icons.m.css';
  *
  * Properties that can be set on a Select component
  *
- * @property CustomOption   Custom widget constructor for options. Should use SelectOption as a base
- * @property describedBy    ID of an element that provides more descriptive text
- * @property disabled       Prevents the user from interacting with the form field
- * @property invalid        Indicates the value entered in the form field is invalid
- * @property label          Label settings for form label text, position, and visibility
- * @property multiple       Whether the widget supports multiple selection
```

Comment:
How is this handled now?

Reply:
Listbox on its own is effectively a custom multiselect widget, and the native multiselect is so universally awful that I don't think we need to go out of our way to support it.